1.1/functions.ttl

```diff
@@ -0,0 +1,27 @@
+PREFIX safuncs: <http://www.opengis.net/def/function/aggregate/geosparql/>
```

Comment:
Can this file be renamed from `functions.ttl` to `sa_functions.ttl` to make it more explicit?

Also, you'll need to add this resource to the Profile declaration, just copy this one:

* <https://github.com/opengeospatial/ogc-geosparql/blob/master/1.1/profile.ttl#L84>